python-zadania/Zadanie-05

```diff
@@ -0,0 +1,14 @@
+KwartalTuple = ("Styczeń", "Luty", "Marzec")
+print(KwartalTuple)
+KwartalTuple[1] = "Kwiecień"
+print(KwartalTuple)
+
+# Przy probie zamiany pierwszego elementu tupli otrzymuję informację, że jest to błędne działanie, 
+# gdyż tupla ne pozwala na zmmianę elementów.
+# Po utworzeniu tupli nie można jej zmienić.
+
+# Poniżej informacja zwrotna z terminala:
+#  File "C:\Users\tinu\Desktop\repozytorium\learn-git\python-zadania\Zadanie-05", line 3, in <module>
+#    KwartalTuple[1] = "Kwiecień"
+#     ~~~~~~~~~~~~^^^
+# TypeError: 'tuple' object does not support item assignment
```

Comment:
Ok, Tupla nie pozwala na zmianę wartosci, kolejnosci ani edycje elementu. Jedyna opcja, którą możemy zrobić to konwersja tupli na listę, odwrócenie typu danych na końcu na tuplę.